ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -218,8 +218,6 @@ public async Task<Notification> GetNotificationWithAllInfoAsync(int notification
                 .Include(n => n.SocialContextAddresses)
                 .Include(n => n.SocialContextVenues).ThenInclude(s => s.VenueType)
                 .Include(n => n.Alerts)
-                .Include(n => n.TreatmentEvents)
-                    .ThenInclude(t => t.TreatmentOutcome)
```

Comment:
Aren't treatment outcomes required for the banner?

Reply:
GetBannerReadyNotificationsIQueryable() already has the treatment outcomes included :)

Reply:
Duh, completely right, I misread this chunk of code as the definition of `GetBannerReadyNotificationsIQueryable`.
As you were :D